contracts/core/Allo.sol

```diff
@@ -13,30 +13,29 @@ import {Clone} from "./libraries/Clone.sol";
 import "./libraries/Native.sol";
 import {Transfer} from "./libraries/Transfer.sol";
 
-/**
- *          ___            ___        ___        ___
- *         /\  \          /\__\      /\__\      /\  \
- *        /::\  \        /:/  /     /:/  /     /::\  \
- *       /:/\:\  \      /:/  /     /:/  /     /:/\:\  \
- *      /::\~\:\  \    /:/  /     /:/  /     /:/  \:\  \
- *     /:/\:\ \:\__\  /:/__/     /:/__/     /:/__/ \:\__\
- *     \/__\:\/:/  /  \:\  \     \:\  \     \:\  \ /:/  /
- *          \::/  /    \:\  \     \:\  \     \:\  /:/  /
- *          /:/  /      \:\  \     \:\  \     \:\/:/  /
- *         /:/  /        \:\__\     \:\__\     \::/  /
- *         \/__/          \/__/      \/__/      \/__/
- */
-
-/// @title Allo
-/// @notice The Allo contract
+/// @title   ___            ___        ___        ___
+///         /\  \          /\__\      /\__\      /\  \
+///        /::\  \        /:/  /     /:/  /     /::\  \
+///       /:/\:\  \      /:/  /     /:/  /     /:/\:\  \
+///      /::\~\:\  \    /:/  /     /:/  /     /:/  \:\  \
+///     /:/\:\ \:\__\  /:/__/     /:/__/     /:/__/ \:\__\
+///     \/__\:\/:/  /  \:\  \     \:\  \     \:\  \ /:/  /
+///          \::/  /    \:\  \     \:\  \     \:\  /:/  /
+///          /:/  /      \:\  \     \:\  \     \:\/:/  /
+///         /:/  /        \:\__\     \:\__\     \::/  /
+///         \/__/          \/__/      \/__/      \/__/
+///
+/// @notice The Allo core contract
+/// @dev This contract is used to create & manage pools as well as manage the protocol. It
+///      is the core of all things Allo.
```

Comment:
`It is the core of all things Allo.`

Does this sentence makes sense?

Reply:
It needs updated.